src/OrchardCore/OrchardCore.DisplayManagement/OrchardCore.DisplayManagement.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\OrchardCore.ContentManagement.Abstractions\OrchardCore.ContentManagement.Abstractions.csproj" />
```

Comment:
Try to avoid this. Maybe you have to create a new Dedicated CssOrchardHelper for your new methods.